index.html

```diff
@@ -43,22 +43,111 @@
+ */
+    #in_person_info {
+      display: none;
+    }
+
+    #volunteer_section {
+      display: none;
+    }
+
+    #badge span {
+      font-family: "Raleway";
+    }
+
+    #badge {
+      position: relative;
+      margin: 0 auto;
+      width: 4in;
+      height: 3in;
+      border: 1px solid black;
+    }
+
+    #badge div {
+      white-space: nowrap;
+      text-align: center;
+      width: 100%;
+      position: absolute;
+      line-height: 0.8;
+    }
+
+    #lname-d,
+    #sname-d {
+      font-weight: bold;
+    }
+
+    #lname-d {
+      font-size: 68pt;
+      bottom: 2.05in;
+    }
+
+    #sname-d {
+      font-size: 26pt;
+      font-weight: bold;
+      bottom: 1.55in;
+    }
+
+    #affili-d,
+    #pronoun-d {
+      font-size: 22pt;
+      bottom: 1.08in;
+    }
+
+    #pronoun-d {
+      font-style: italic;
+      bottom: 0.64in;
+    }
+
+    #event-d,
+    #location-d {
+      font-size: 7pt;
+      bottom: 0.29in;
+      left: 0.025in;
+    }
+
+    #location-d {
+      bottom: 0.15in;
+    }
+
+    #logo1 {
+      width: 92px !important;
+      height: 61px;
+      background-image: url("https://raw.githubusercontent.com/LSSTDESC/name-tag-guru/master/web-static/desc-logo.png");
+      bottom: 0.02in;
+      left: 0.1in;
+    }
 
+    #logo2 {
+      width: 64px !important;
+      height: 22px;
+      background-image: url("https://raw.githubusercontent.com/LSSTDESC/name-tag-guru/master/web-static/lsstc-logo.png");
+      bottom: 0.15in;
+      right: 0.2in;
+    }
   </style>
 
 </head>
 
 <body>
   <div class="container">
-    <h2>Registration for the February 2024 DESC Virtual Meeting</h2>
-    <h4>Monday - Friday, February 26th - March 1st, 2024</h4>
-    There is no registration fee for this meeting.
+    <h2>Registration for the July 2025 DESC Collaboration Meeting</h2>
+    <h4>Monday - Friday, July 21 - 25, 2025</h4>
+    There is an in-person registration fee of $170 for early
+    career participants (students and postdocs) and $200 for senior participants at the main (University of Illinois) site. There is no registration fee for
+    satellite sites or remote participation. The fee
+    includes venue and other miscellaneous costs.  Registration for in-person attendance will
+    close on July DD.
     <br>
     Please join the LSSTC Slack channel <a href="https://lsstc.slack.com/archives/C90AMAG80"
       target="_blank" rel="noopener noreferrer">#desc-collab-meeting</a> to keep updated on chats, discussions, and announcements during the
     meeting.
     <br>
-      You can find the details of the schedule on this <a href="https://confluence.slac.stanford.edu/x/O4TqG" target="_blank" rel="noopener noreferrer">confluence page</a> as they become available.
+      You can find the details of the schedule on
+      <!-- this <a href="https://confluence.slac.stanford.edu/x/O4TqG" target="_blank" rel="noopener noreferrer">confluence page</a> -->
+      some confluence page as they become available.
```

Comment:
Suggest https://confluence.slac.stanford.edu/x/rJ6QIw (the main page for the July 2025 DESC meeting, which has links to the developing agenda)